### ISSUE TITLE
many open instances

### ISSUE THREAD

**Issue by barnosch** [nebulx29/MMM-dht22#1]
Hi. I have many open instances (more then 50) of the script. Can i do something against it?
It works everything and the pi is restarted on a regular basis
![image](https://user-images.githubusercontent.com/31805456/31861823-dd7efcb4-b733-11e7-9e29-7ec1e5028846.png)

**Comment by nicholascowan** [comment]
Same issue.

**Comment by herbyFIT** [comment]
how did you setupt your DHT sensor with this module?

**Comment by sherwood65633** [comment]
Yes, I get nothing but "reading.....". dht22 util is setup in bin folder..and pins are right...Anything i am missing?
